src/main/blackbox/blackbox.c

```diff
@@ -890,12 +905,12 @@ static void writeInterframe(void)
 
 #ifdef USE_SERVOS
     if (testBlackboxCondition(CONDITION(SERVOS))) {
-        STATIC_ASSERT(ARRAYLEN(servo) <= 8, "TAG8_8SVB supports at most 8 values"); 
```

Comment:
(This will cause merge conflict with your other BB related PR)

Reply:
> (This will cause merge conflict with your other BB related PR)

The servo array length is defined in src\main\target\common_defaults_post.h
The servo length can be 1 or 8.
Yes, if it will > 8, the using of TAG8_8SVB will unpossible

Reply:
It's whitespace-only change. But IIRC you are fixing WS in multiple PRs, that will confuse git after merging first one

Reply:
Seems i understand:) Are you mean unnecessary white spaces?
Yes, our reviewers learn me! :)
I use magic keys in my text editor to save without undue white spaces.
But github shows that all are good in these cases usually.

Reply:
Seems i understand:) Are you mean unnecessary white spaces?
Yes, our reviewers learn me! :)
I use magic keys in my text editor to save without undue white spaces.
But github shows that all are good in these cases usually.

Reply:
yes, the problem is that the code is not changed, but your editor cleaned up whitespace.  rather than allowing this in your PR, it should be a separate cleanup/refactor PR.

Reply:
@demvlad : Yes. Don't worry about it. But you'll have to resolve conflicts in one of the PRs eventually (github will display the error)

@nerdCopter : 
It would be great to normalize whole codebase and add hook that will prevent trailing whitespace ...
But for small whitespace changes close to edited code, it is IMO fine to fix it ...
(and it is so annoying with trailing whitespace highlighting)

Reply:
>  I use magic keys in my text editor to save without undue white spaces. 

But there are no magic key to remove myself white space only....